tldr/hybrid_extractor.py

```
@@ -3151,6 +3224,315 @@ def _extract_elixir_function(self, node, source: bytes, is_private: bool) -> Fun
+
+            # Recurse into all children
+            self._extract_php_calls(child, caller_name, source, call_graph, module_info, class_bases, _fn_names, _ns)
+
+    def _get_php_member_call_name(self, node, source: bytes) -> str | None:
+        """Get the method name from a member_call_expression ($this->method())."""
+        for child in node.children:
+            if child.type == "name":
+                return self._safe_decode(source[child.start_byte:child.end_byte])
+        return None
+
+    def _get_php_scoped_call_name(self, node, caller_name: str | None, source: bytes, module_info: "ModuleInfo | None" = None, class_bases: list[str] | None = None) -> str | None:
```

Ruff (0.15.7)

[warning] Too many branches (17 > 12)

(PLR0912)